src/create-release/index.ts

```diff
@@ -11,6 +11,12 @@ export async function actionCreateRelease() {
     const releaseVersion = core.getInput('version')
     const releaseNotes = core.getInput('release-notes')
     const apiEndpoint = core.getInput("replicated-api-endpoint") || process.env.REPLICATED_API_ENDPOINT;
+    const parsedTimeout = parseInt(core.getInput('timeout-minutes') || '20');
```

Comment:
This should also be added to action.yml

Reply:
thank you! I have updated action.yml and README.md